docs/section11/Rule11-15.md

```diff
@@ -0,0 +1,48 @@
+# ServiceWaterHeating - Rule 11-15
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-15  
+
+**Rule Description:** "Service water loads and use shall be the same for both the proposed design and baseline building design and shall be documented by the calculation procedures described in Section 7.4.1.
+Exceptions: 
+(1) ECMs are used to reduce SWH use.. 
+(2) SWH energy consumption can be demonstrated to be reduced by reducing the required temperature of service mixed water, by increasing the temperature, or by increasing the temperature of the entering makeup water. "  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (g)  
+
+**Evaluation Context:** Each SWH use  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_component_by_id()
+
+**Applicability Checks:**
+- Each SHW use is applicable
```

Comment:
Seems like these `for` loops should be a part of the rule logic. The applicability check should be that the building has at least 1 SHW use (where ServiceWaterHeatingUse.use > 0?)